### PR TITLE
feat: add size to entry

### LINKED ISSUE #316: Add Size to Entry

### PR DESCRIPTION
**Pull request by kaitoz11**

### CODE REVIEW

**Comment by ChienNQuang** [comment]
it fails unit test

**Comment by ChienNQuang** [comment]
nvm its my fault from before

File: src/Application/Digital/Commands/UploadDigitalFile.cs [hunk 1]
```diff
@@ -89,6 +90,8 @@ public async Task<EntryDto> Handle(Command request, CancellationToken cancellati
             }
             else
             {
+                
+                entryEntity.Size = request.FileData!.Length;
```
Comment (ChienNQuang) [inline src/Application/Digital/Commands/UploadDigitalFile.cs]: why don't move this down to centralize updating an entity?

File: src/Domain/Entities/Digital/Entry.cs [hunk 2]
```diff
@@ -9,6 +9,7 @@ public class Entry : BaseAuditableEntity
     public string Path { get; set; } = null!;
     public Guid? FileId { get; set; }
     public Guid OwnerId { get; set; }
+    public long? Size { get; set; }
```
Comment (ChienNQuang) [inline src/Domain/Entities/Digital/Entry.cs]: what unit is size? how do I know?

**Review by ChienNQuang — CHANGES_REQUESTED** [review]
how do I know if size is in MB or KB?

File: src/Domain/Entities/Digital/Entry.cs (hunk 2 above)
Comment (kaitoz11) [reply src/Domain/Entities/Digital/Entry.cs]: fixed at https://github.com/ProFile-org/ProFile-BE/pull/315/commits/80c9fb7b5afe06e2373c313e26bf54dd9fdf7406

File: src/Application/Digital/Commands/UploadDigitalFile.cs (hunk 1 above)
Comment (kaitoz11) [reply src/Application/Digital/Commands/UploadDigitalFile.cs]: fixed at https://github.com/ProFile-org/ProFile-BE/pull/315/commits/80c9fb7b5afe06e2373c313e26bf54dd9fdf7406

**Review by ChienNQuang — APPROVED** [review]
sure

**Pull request merged**